charts/enterprise-kyverno-operator/templates/NOTES.txt

```diff
@@ -2,6 +2,23 @@ Chart version: {{ .Chart.Version }}
 
 Thank you for installing {{ .Chart.Name }}! Your release is named {{ .Release.Name }}.
 
+{{- if .Values.enablePolicyset }}
+{{- $policyCRD := lookup "apiextensions.k8s.io/v1" "CustomResourceDefinition" "" "policies.kyverno.io" }}
+{{- $clusterPolicyCRD := lookup "apiextensions.k8s.io/v1" "CustomResourceDefinition" "" "clusterpolicies.kyverno.io" }}
+{{- if or (not $policyCRD) (not $clusterPolicyCRD) }}
+********************************************************************************
+⚠️  WARNING: Required Kyverno CRDs are missing
+--------------------------------------------------------------------------------
+The policyset controller requires the following Kyverno CRDs to function:
+
+- Policy (`policies.kyverno.io`)
+- ClusterPolicy (`clusterpolicies.kyverno.io`)
+
+❗ These CRDs are not currently installed in the cluster.
+   Please install Kyverno or ensure its CRDs are present before using this chart.
+{{- end }}
+{{- end }}
+
 💡 Note: There is a trade-off when deciding which approach to take regarding Namespace exclusions. Please see the documentation at https://kyverno.io/docs/installation/#security-vs-operability to understand the risks.
```

Comment:
Remove this Note section. Its not relevant anymore ever since we decoupled kyverno deployment from the operator.